lib/scrabble.rb

```diff
@@ -1,7 +1,31 @@
 class Scrabble
 
   def score(word)
-    1
+    word = usable_word(word)
+    single_multipliers = Array.new(word.length, 1)
+    score_with_multipliers(word, single_multipliers)
+  end
+
+  def score_with_multipliers(word, letter_multiplers, word_multiplier = 1)
+    word = usable_word(word)
+
+    score = word.each_char.with_index.reduce(0) do |points, (letter, index)|
+      points + letter_score(letter) * letter_multiplers[index]
+    end
```

Comment:
Zipping is probably the way you want to go here. Indexes are all over the place in other languages, but they're not very idomatic in Ruby. I also have moved towards `.map` and `.sum` over `.reduce` in these situations now that `.sum` is native in Ruby 2.4. 

```
letter_multiplier_pairs = word.each_char.zip(letter_multipliers)
score = letter_multiplier_pairs.map do |(letter, multiplier)|
  letter * multiplier
end.sum
```

If we were able to sit down and have a proper code review, I'd give you the chance to defend your decision. Take this as a suggestion. Your way isn't necessarily wrong if you think it's more expressive and easier to read. These are the things to care about.